README.md

```diff
@@ -1105,16 +1105,16 @@ module Authentication
   ...
   def remember(user)
     user.regenerate_remember_token
-    cookies.permanent.encrypted[:remember_token] = user.remember_token
+    cookies.encrypted[:remember_token] = user.remember_token
```

Comment:
Wouldn't we want to keep the call to `permanent` here, since we're setting the value, and not reading from it?

Reply:
Indeed, fixed 👍